pkg/manifests/manifests.go

```diff
@@ -1420,7 +1424,10 @@ func (f *Factory) PrometheusK8s(host string, grpcTLS *v1.Secret, trustedCABundle
 	}
 
 	p.Spec.Image = &f.config.Images.Prometheus
-	p.Spec.ExternalURL = f.PrometheusExternalURL(host).String()
+
+	if f.consoleConfig != nil {
+		p.Spec.ExternalURL = f.consoleConfig.Status.ConsoleURL + "/monitoring"
```

Comment:
(nit) 
```suggestion
		p.Spec.ExternalURL = path.Join(f.consoleConfig.Status.ConsoleURL, "monitoring")
```

Reply:
:+1: Makes sense yes, any objections against a followup PR? The manifest creation for Alertmanager has the same code and could use the same fix.

Reply:
not at all